frontend/public/components/cloud-services/clusterserviceversion-resource.tsx

```diff
@@ -186,10 +186,8 @@ export const ClusterServiceVersionResourceDetails = connectToPlural(
         </div>
         <div className="co-m-pane__body">
           <div className="co-clusterserviceversion-resource-details__section co-clusterserviceversion-resource-details__section--info">
+            <CompactExpandButtons expand={this.state.expanded} onExpandChange={(expanded) => this.setState({expanded})} className="co-clusterserviceversion-resource-details__compact-expand" />
```

Comment:
It feels a little weird to pass a `className` through in this way, but I don't have a great suggestion on how to better do it. @ggreer any ideas?

Reply:
It's not uncommon in React, but it doesn't seem like the right approach here to pollute the interface. Can we just add a wrapper `div` and apply the styles to it?

Reply:
A wrapper will require CSS rules that utilize descendant selectors to get the styling right, which it seems like y'all try to avoid.  I assumed this was the lesser of two evils?

Reply:
Changing the markup to this and using the CSS class you wrote yields the same result:
```html
<div className="co-clusterserviceversion-resource-details__compact-expand">
  <CompactExpandButtons expand={this.state.expanded} onExpandChange={(expanded) => this.setState({expanded})} />
</div>
<div className="co-clusterserviceversion-resource-details__section co-clusterserviceversion-resource-details__section--info">
  ...
</div>
```

Reply:
Ah!  Thanks for the clarification, @alecmerdler.  I see what you mean now.  Will make recommended change.